content/docs/user-guide/contributing/docs.md

```diff
@@ -59,6 +59,15 @@ formatted and linted as well, which is also ensured by the full setup below.
 Make sure you have a recent LTS version of [Node.js](https://nodejs.org/en/)
 (`>=12.0.0`, `<=15.x`), and install [Yarn](https://yarnpkg.com/):
 
+> In Windows, you may need to install [Python], [Visual Studio Build Tools], and
+> [Windows SDK] additionally for dependencies.
```

Comment:
```suggestion
> [Windows SDK] first.
```

Reply:
Why would the Windows SDK be needed though? Seems overkill, isn't it several GBs?

Reply:
I remember some package in the requirements asking for it. Some of these are compiled packages and probably need one or two functions from the SDK. Seems overkill, yes.